cmake/hdcam.cmake

```diff
@@ -7,20 +7,27 @@
 # folders should be on the system path.
 find_path(DCAMSDK_ROOT_DIR
     NAMES "dcamsdk4/inc/dcamapi4.h"
-    PATH_SUFFIXES "Hamamatsu_DCAMSDK4_v22126552"
+    PATH_SUFFIXES 
```

Comment:
@nclack @aliddell , are we ok adding another possible version of the SDK?  It isn't readily apparent how to download older versions from their website.

Reply:
I think it's probably worth forcing the upgrade to just use the latest. Or maybe there's a way to glob on this?

Reply:
Lets upgrade. Not sure what you mean re glob @aliddell. Let me know what version is best and I'll install it on the test machine.